backends/arm/tosa_quant_utils.py

```diff
@@ -196,7 +196,15 @@ def build_rescale_conv_output(
     post_conv2d_scale = (input_scale.number * weight_scale.number) / output_scale.number
 
     # Since we assume the input tensor that is being rescaled is int32 date type, zero point must be 0.
-    rescale_op = build_rescale(
-        tosa_fb, post_conv2d_scale, op, output_type, op.shape, 0, 0, double_round
+    build_rescale(
```

Comment:
Unrelated to NHWC changes?

Reply:
yes, it's not very related but as a code improvement for conv2d which is used in MNV2. I would suggest to leave it here (same reasons above)